src/utils.js

```diff
@@ -0,0 +1,195 @@
+class Task {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const restoreOriginalIcons = () => {
+  const iconDiv = document.querySelectorAll('.icon');
+  iconDiv.forEach((icon) => {
+    icon.innerHTML = '&#8942';
+    icon.style.cursor = 'all-scroll';
+  });
+};
+
+const restoreBackgroundWhite = () => {
+  const labelTasks = document.querySelectorAll('li');
+  labelTasks.forEach((label) => {
+    label.style['background-color'] = '#fff';
+  });
+};
+
+const scrapeAndSave = () => {
+  const tasksArray = [];
+  const tasks = document.querySelectorAll('.task');
+  tasks.forEach((label, index) => {
+    const parentTaskCont = label.parentNode;
+    const checkboxElem = parentTaskCont.querySelector('.checkBoxesTasks');
+    const task = new Task(label.innerHTML, checkboxElem.checked, index + 1);
+    tasksArray.push(task);
+  });
+  localStorage.removeItem('tasksList');
+  localStorage.setItem('tasksList', JSON.stringify(tasksArray));
+};
+
+function setValueTextArea(textArea, labelTask) {
+  labelTask.innerHTML = textArea.value;
+  textArea.style.display = 'none';
+  labelTask.style.display = 'block';
+  restoreOriginalIcons();
+  restoreBackgroundWhite();
+  scrapeAndSave();
+}
```

Comment:
Great job with refactoring your code 🥳 
OPTIONAL:
I would suggest that you make all your functions into arrow function.
In this case specifically, you can use the sintax: `const setValueTextArea = (textArea, labelTask) => { YOUR CODE HERE }`